variables.tf

```diff
@@ -121,6 +121,11 @@ variable "materialize_instances" {
     balancer_memory_limit   = optional(string, "256Mi")
     balancer_cpu_request    = optional(string, "100m")
     license_key             = optional(string)
+    environmentd_extra_env = optional(list(object({
+      name  = string
+      value = string
+    })), [])
+    environmentd_extra_args = optional(list(string), [])
```

Comment:
I think we should provide some example for using this?

Reply:
Yes! I was also thinking about this, do you have any specific args in mind that you would like us to showcase in the examples?

Reply:
Just added some more info about the new options in the readme file, let me know what you think! Open to suggestions on the specific values!